compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposePanel.desktop.kt

```diff
@@ -20,9 +20,11 @@ import androidx.compose.runtime.CompositionLocalProvider
 import androidx.compose.ui.ExperimentalComposeUiApi
 import androidx.compose.ui.focus.FocusDirection
 import androidx.compose.ui.window.WindowExceptionHandler
+import androidx.compose.ui.window.layoutDirectionFor
 import java.awt.*
 import java.awt.event.FocusEvent
 import java.awt.event.FocusListener
+import java.util.*
```

Comment:
Check your IDEA style, it shouldn't allow "*" imports now (the style was merged 2 month ago).

<img width="458" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/e6a96d5b-63cc-4242-b837-50b62e9ef7cd">

Reply:
I have that, but IDEA insists on importing `java.util.*` for `Locale` (`java.awt.*` was already there).

Reply:
Must be this:

<img width="253" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5230206/fc8797f0-30e9-48f9-9961-ad250b9c1614">

Reply:
I also have it, but it is disable in my config 🤔
<img width="531" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/1ee95ae6-7513-41de-ba0f-ec3f55c10fca">